usaspending_api/search/v2/views/search.py

```diff
@@ -344,3 +346,60 @@ def post(self, request):
 
             else:  # recipient_type
                 raise InvalidParameterException('recipient type is not yet implemented')
+
+
+class SpendingByGeographyVisualizationViewSet(APIView):
+
+    def post(self, request):
+        """Return all budget function/subfunction titles matching the provided search text"""
+        json_request = request.data
+        scope = json_request.get('scope', None)
+        filters = json_request.get('filters', None)
+
+        if scope is None:
```

Comment:
This should be combined into a single if statement as such: `if not scope or not filter` which can be further simplified to `if not (scope and filter)`.

Reply:
Decided we're leaving it as is to be specific for error handling